src/requestResponse.js

```diff
@@ -8,7 +8,7 @@ const pruneRequestTimeout = 60 * 60 * 1000
 const pruneIntervalRate = 15 * 60 * 1000
 let pruneInterval
 
-function createRequest(type, clientRequest, user, clientIp, updateCb) {
+function createRequest(app, type, clientRequest, user, clientIp, updateCb) {
```

Comment:
I would have passed just `intervals`, easier to understand what `createRequest`'s real dependencies are.